### PR TITLE
Add devtoolset-7

### PR DESCRIPTION
**Pull request by hhorak**

### CODE REVIEW

File: 7-toolchain/Dockerfile.rhel7 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Developer Toolset 7 Toolchain"
+
+# We need to call 2 (!) yum commands before being able to enable repositories properly
+# This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1479388
+RUN yum repolist > /dev/null && \
+    yum install -y yum-utils && \
+    yum-config-manager --disable \* &> /dev/null && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-server-rhscl-beta-7-rpms \
```
Comment (omron93) [inline 7-toolchain/Dockerfile.rhel7]: @hhorak `rhel-server-rhscl-7-beta-rpms` should be there.

File: 7-perftools/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
+FROM centos:centos7
+LABEL MAINTAINER Red Hat, Inc.
+
+ENV SUMMARY="Red Hat Developer Toolset 7 Performance Tools container image" \
+    DESCRIPTION="Performance tools: systemtap, valgrind, dyninst, elfutils, oprofile"
+
+LABEL com.redhat.component="devtoolset-7-perftools-docker" \
+      name="centos/devtoolset-7-perftools-centos7" \
+      version="7" \
```
Comment (pkubatrh) [inline 7-perftools/Dockerfile:9]: Can we start the versioning at `1`, same as the other new 3.0 images?

Comment (pkubatrh) [reply 7-perftools/Dockerfile:9]: Plus dropping the release label

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
> Can we start the versioning at 1, same as the other new 3.0 images?

Version 7 was already used for Beta images, so I'm afraid it could be misleading for end users to change later. Also, I don't think it must be the same for all images, the important thing for us is to be able to bump when we need, which is possible even from 7.. So, I'm more for keeping the 7 and not bother changing it..

**Comment by hhorak** [comment]
The CentOS test also passes, although due to not-yet-accepted invite of centos-ci user it is not shown here. https://ci.centos.org/view/SCLo-images/job/SCLo-container-devtoolset-rh/78/consoleFull

**Comment by pkubatrh** [comment]
>Version 7 was already used for Beta images, so I'm afraid it could be misleading for end users to change later.

Makes sense. LGTM now

**Pull request merged**